package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "aasvg": "0.3"
```

Comment:
I can also do "0.3.x" to be more explicit. My reasoning (here, and in other software projects) is that any patch version will work (assuming semver). Any new checkout can just use the latest 0.3.x. On the other hand pinning an explicit patch version just means that you have to update, commit, and push it more often to stay up-to-date. And if you want that level of detailed control, you might as well commit the lock file.

Reply:
You're all good, this is a personal preference thing. You actually made me read the `package.json` docs on this since I actually had never done it this way, and you are right (this is why I said "I guess it is the same thing") and approved also. I guess I can't "resolve comment" on this repo/org, but you can safely ignore this then, it will get merged shortly I have no doubt.

(Sorry for the cross wires before, I am not sure how I conflated different issues in your very clear PR, but oh well, next time I will read better.)